shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -3446,22 +3485,24 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
 
     private newtype TPathNode =
       TPathNodeMid(
-        NodeEx node, FlowState state, CallContext cc, SummaryCtx sc, DataFlowType t, AccessPath ap
+        NodeEx node, FlowState state, CallContext cc, SummaryCtx sc, DataFlowType t, AccessPath ap,
+        string summaryLabel
```

Comment:
That's for summaries. Otherwise we won't actually see the provenance information from a synthesised MaD method body when it's summarised as a single edge at the call site.